Makefile

```diff
@@ -1,7 +0,0 @@
-DOCS_OUTPUT_DIR ?= generated/docs
```

Comment:
I'm not sure if anyone was relying on this to test docs locally. If we don't have it already, can we add a short section in the dev docs on how to build/view docs locally?